src/main/java/domain/Results.java

```diff
@@ -0,0 +1,44 @@
+package domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Results {
+
+    private final List<Result> results;
+
+    private Results(List<Result> results) {
+        this.results = results;
+    }
+
+    public static Results of(String[] result, int numberOfPlayers) {
+        List<Result> results;
+        results = Arrays.stream(result)
+                .map(Result::new)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        List<Result> results = Arrays.stream(result)
                .map(Result::new)
                .collect(Collectors.toList());
```

초기화와 선언을 한번에 하면 더 직관적입니다-!

Reply:
반영했습니다! 꼼꼼한 피드백 감사합니다!!